raft/rafttest/node.go

```diff
@@ -79,6 +108,10 @@ func (n *node) start() {
 				}
 				n.Advance()
 			case m := <-n.iface.recv():
+				// previous leader failure
+				if m.Type == raftpb.MsgProp && n.id == n.lead && lead == 0 {
```

Comment:
i do not quite understand this. n.id == n.lead -> n is the leader. what is lead==0?